migration/ods1_stage/tables/base/address/spu_translated_address.sql

```diff
@@ -0,0 +1,96 @@
+--- 1. etl_spuMergeOfficeAddress (snowflake db)
+
+-- CTE 'w' retrieves records from FacilityProfileProcessing and FacilityProfileProcessingDeDup tables
+-- It also extracts 'Address' from 'EntityJSONString' and assigns it to the 'FacilityJSON' column
+WITH cte_w AS (
+    SELECT 
+        p.CREATE_DATE, 
+        p.RELTIO_ID as ReltioEntityID, 
+        p.Facility_CODE as FacilityCode, 
+        p.FacilityID,
+        PARSE_JSON(p.PAYLOAD:EntityJSONString.Address) AS FacilityJSON  -- Extract 'Address' from 'EntityJSONString' !!!!!!!!!!!!!!!!! THIS MIGHT BE INCORRECT
+    FROM raw.FacilityProfileProcessingDeDup AS d
+    JOIN raw.FacilityProfileProcessing AS p ON p.rawFacilityProfileID = d.rawFacilityProfileID
+    WHERE p.PAYLOAD IS NOT NULL
+),
+
+-- CTE 'x' retrieves all records from 'w' where 'FacilityJSON' is not NULL
+cte_x AS (
+    SELECT *
```

Comment:
Let's avoid the select *. Please add the columns you will need here.
FacilityID
FacilityJSON
FacilityCode
CREATE_DATE